renovate.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": [
+    "config:base",
+    "config:semverAllMonthly"
+  ],
+  "pinVersions": false,
```

Comment:
[`config:semverAllMonthly`](https://github.com/renovateapp/renovate-config/blob/master/packages/renovate-config-config/package.json#L60) has [`"pinVersions": false`](https://github.com/renovateapp/renovate-config/blob/master/packages/renovate-config-default/package.json#L43) already.
Do we still need this option?